onnxruntime/test/onnx/main.cc

```diff
@@ -683,7 +740,16 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
         ORT_TSTR("sce_NCd1d2d3_sum_weight_high_ii"),
         ORT_TSTR("sce_NCd1d2d3_sum_weight_high_ii_expanded"),
         ORT_TSTR("sce_none_weights_log_prob_expanded"),
-        ORT_TSTR("sce_none_weights_expanded")};
+        ORT_TSTR("sce_none_weights_expanded"),
+        // onnx==1.14.0
+        ORT_TSTR("loop13_seq"),
+        ORT_TSTR("maxunpool_export_with_output_shape"),
+        ORT_TSTR("resize_downsample_scales_cubic_align_corners"),
+        ORT_TSTR("resize_downsample_scales_linear_align_corners"),
+        ORT_TSTR("training_dropout"),
+        ORT_TSTR("training_dropout_default"),
+        ORT_TSTR("training_dropout_default_mask"),
+        ORT_TSTR("training_dropout_mask")};
```

Comment:
so these are only failing for QNN EP?

Reply:
I removed them from the list. They were mentioned in the output.  I started this PR 2 months ago when onnx was far from being released. They might have appeared at one stage.